src/data.js

```diff
@@ -1,24 +1,33 @@
 import compact from "lodash/array/compact";
 import findIndex from "lodash/array/findIndex";
 import flatten from "lodash/array/flatten";
-import take from "lodash/array/take";
 import union from "lodash/array/union";
 import isEmpty from "lodash/lang/isEmpty";
+import isFunction from "lodash/lang/isFunction";
+import isUndefined from "lodash/lang/isUndefined";
+import isNull from "lodash/lang/isNull";
+import has from "lodash/object/has";
+import assign from "lodash/object/assign";
 import merge from "lodash/object/merge";
+import identity from "lodash/utility/identity";
+import property from "lodash/utility/property";
 import lodashRange from "lodash/utility/range";
 import uniq from "lodash/array/uniq";
 import zipObject from "lodash/array/zipObject";
-import zip from "lodash/array/zip";
-import * as Collection from "./collection";
 import * as Style from "./style";
 import Scale from "./scale";
 
-module.exports = {
+export default {
   // String Data
-  createStringMap(props, axis) {
+  createStringMap(props, axis, hasMultipleDatasets = false) {
```

Comment:
I could, since `undefined` is falsey, but I generally like using default parameters like this to clarify that the prop is optional (and not always passed), and so that it's always a boolean. LMK if you want to avoid this convention.